tests/integration/common/checks-after-conversion/test_check_repoquery_empty_exclude.py

```diff
@@ -0,0 +1,10 @@
+def test_check_repoquery_empty_exclude(shell):
+    """
+    Verify that convert2rhel used `--setopt=exclude=` in every `repoquery` call.
+    """
+    log_file = "/var/log/convert2rhel/convert2rhel.log"
+    number_of_repoquery_calls = shell(f"grep repoquery {log_file} | wc -l").output.strip()
+    number_of_repoquery_calls_with_exclude = shell(
+        f'grep "repoquery.*--setopt=exclude=\s.*" {log_file} | wc -l'
+    ).output.strip()
+    assert int(number_of_repoquery_calls) == int(number_of_repoquery_calls_with_exclude)
```

Comment:
Any reason why this is not included here? https://github.com/oamg/convert2rhel/blob/main/tests/integration/common/checks-after-conversion/test_verify_strings_in_log.py

Reply:
404: reason not found